sklearn/model_selection/_validation.py

```diff
@@ -318,7 +318,7 @@ def cross_val_predict(estimator, X, y=None, labels=None, cv=None, n_jobs=1,
         Group labels for the samples used while splitting the dataset into
         train/test set.
 
-    cv : int, cross-validation generator or an iterable, optional
+    cv_generator : int, cross-validation generator or an iterable, optional, default: None
```

Comment:
Huh? Why this rename?

Also PEP8.

And default is effectively 3, so None is confusing, IMO.